fuzzit.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+set -xe
+
+# first argument must be:
+# fuzzing          : upload to fuzzit.dev servers for continuous fuzzing
+# local-regression : run a short regression locally using past crashing test cases
+
+# go-fuzz doesn't support modules yet, so ensure we do everything
+# in the old style GOPATH way
+export GO111MODULE="off"
+
+# install go-fuzz
+go get -u github.com/dvyukov/go-fuzz/go-fuzz github.com/dvyukov/go-fuzz/go-fuzz-build
+
+# target name can only contain lower-case letters (a-z), digits (0-9) and a dash (-)
+# to add another target, make sure to create it with `fuzzit create target`
+# before using `fuzzit create job`
+TARGETS=("promql-parse-metric" "promql-parse-open-metric" "promql-parse-metric-selector" "promql-parse-expr")
+
+go-fuzz-build -libfuzzer -func FuzzParseMetric -o promql-parse-metric.a ./promql
+go-fuzz-build -libfuzzer -func FuzzParseOpenMetric -o promql-parse-open-metric.a ./promql
+go-fuzz-build -libfuzzer -func FuzzParseMetricSelector -o promql-parse-metric-selector.a ./promql
+go-fuzz-build -libfuzzer -func FuzzParseExpr -o promql-parse-expr.a ./promql
+
+for TARGET in "${TARGETS[@]}"
+do
+    clang -fsanitize=fuzzer ${TARGET}.a -o ${TARGET}
+done
```

Comment:
Does it make better sense to write it in a single loop?

```sh
TARGETS=("FuzzParseMetric" "...")
for TARGET in "${TARGETS[@]}"
do
    go-fuzz-build -libfuzzer -func "$TARGET" -o "$TARGET.a" ./promql
    clang -fsanitize=fuzzer "$TARGET".a -o "$TARGET"
    rm "$TARGET".a
done
```

Reply:
Yes this would be better but the problem is that the name of targets on fuzzit and the function names are not the same. i.e -func $TARGET won't work. maybe I can do another array with FUZZ_FUNCTIONS?

Reply:
@msiebuhr  I change it to one loop, let me know if you think this is better?